org.kde.vvave.json

```diff
@@ -0,0 +1,101 @@
+{
+    "id": "org.kde.vvave",
+    "rename-icon": "vvave",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "vvave",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--own-name=org.mpris.MediaPlayer2.vvave",
+        "--socket=pulseaudio"
+    ],
+    "separate-locales": false,
```

Comment:
Is there some reason not to separate locales?

Reply:
not really, I just copied the base manifest from another manifest that had this line.